### PR TITLE
search: spacing in journal titles

### PR DESCRIPTION
**Pull request by nooraangelva**
Ref: https://github.com/inspirehep/inspirehep/issues/2359

### CODE REVIEW

File: inspire_query_parser/visitors/elastic_search_visitor.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ class ElasticSearchVisitor(Visitor):
     JOURNAL_ART_ID = 'artid'
     JOURNAL_YEAR = 'year'
     JOURNAL_FIELDS_MAPPING = {
-        JOURNAL_TITLE: '.'.join((JOURNAL_FIELDS_PREFIX, JOURNAL_TITLE, 'raw')),
+        JOURNAL_TITLE: ('journal_title_variants'),
```
Comment (drjova) [inline inspire_query_parser/visitors/elastic_search_visitor.py:80]:
```suggestion
        JOURNAL_TITLE: 'journal_title_variants',
```

File: tests/test_elastic_search_visitor.py [hunk 2]
```diff
@@ -309,13 +309,14 @@ def test_elastic_search_visitor_find_exact_author_with_bai_partial_value_ellis()
...
 def test_elastic_search_visitor_find_journal_title_simple_value():
+
     query_str = 'j Phys.Lett.B'
     expected_es_query = \
         {
             "nested": {
                 "path": "publication_info",
                 "query": {
-                    "match": {"publication_info.journal_title.raw": "Phys.Lett.B"}
+                    "match": {"journal_title_variants": "Phys.Lett.B"}
```
Comment (MJedr) [inline tests/test_elastic_search_visitor.py:319]: I think that's incorrect since now `journal_title_variants` is outside `publication_info` field.

Comment (nooraangelva) [reply tests/test_elastic_search_visitor.py:318]: I think you are right, but the ```_parse_query(query_str)``` still seems to add the ```publication_info```.  Would the correct ```path``` be then ```source``` or an empty ```path```? I will try to find the piece of code that still adds the ```publication_info``` to the ```path``` in the code.

Comment (MJedr) [reply tests/test_elastic_search_visitor.py:318]: It's not about the path only, it's about how we create journal queries https://github.com/inspirehep/inspire-query-parser/blob/bce7704864be3b493a986ae8e9c4b30133fd4353/inspire_query_parser/visitors/elastic_search_visitor.py#L514

File: inspire_query_parser/utils/visitor_utils.py [hunk 3]
```diff
@@ -432,6 +432,8 @@ def generate_nested_query(path, queries):
     Returns:
         (dict): The nested query if queries is not falsy, otherwise an empty dict.
     """
+    import pdb
+    pdb.set_trace()
```
Comment (drjova) [inline inspire_query_parser/utils/visitor_utils.py:436]: oups 👽

**Pull request closed without merging**